python/plugins/processing/algs/qgis/PointsToPaths.py

```diff
@@ -1,5 +1,4 @@
 # -*- coding: utf-8 -*-
-
```

Comment:
Can you re-insert the removed spacing?

Some of the removed empty lines are required

Reply:
done

Reply:
I think this specific line is still missing?